src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_NUMBER = 100;
+    private static Type type;
+
+    public FigureSupplier(Type type) {
+        this.type = type;
+    }
+
+    public static Figure getRandomFigure() {

```

Comment:
```suggestion
    public Figure getRandomFigure() {

```

it is better to avoid using `static` https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5